### ISSUE TITLE
[Bug] Duplicate imports

### ISSUE THREAD

**Issue by dspacejs** [geigi/cozy#439]
### Bug description

Duplicate tracks are being loaded, despite there only being a single file for each track:

![Screenshot from 2021-02-24 12-50-50](https://user-images.githubusercontent.com/9282837/108937860-00c94480-769f-11eb-9b8f-b273a0a5436f.png)

### Steps to reproduce

1. Add audiobooks to an internal drive
2. Import them into Cozy
3. Quit Cozy
4. Unmount the drive
5. Open Cozy again
6. There should be duplicate tracks/audiobooks (the first time I opened it, there were duplicate audiobooks. After reopening, there were only duplicate tracks)

### System Information
- Operating System: Ubuntu 20.04.1 LTS
- Installation source: flatpak
- Version of cozy: 0.8.1

**Comment by dspacejs** [comment]
@geigi do you have a workaround for this? I tried a bunch of things, like removing the imported audiobooks and reimporting them, but this didn't work.

**Comment by geigi** [comment]
Sorry for the late answer, I'm a bit busy at the moment...
The only thing I can currently think of:
Have you made sure you have added the storage location only once in settings? I'm not sure what happens when you add the same location twice. 

If that isn't the case:
What would help with debugging is having a look at Cozy's database (e.g. https://sqlitebrowser.org)

Cozy has different tables and the error can be most likely detected there.

You find the database here (for flatpak)
```
~/.var/app/com.github.geigi.cozy/data/cozy/cozy.dB
```

I think cozy's database is not complicated if you have worked with a SQLite database before. Sharing a database with me for debugging may be not ok with everybody as it contains all the information about the media you've imported into your library (album art, paths, names...)

Let me know if you've added the storage location only once, we then have to think about a way on how to debug this :)

**Comment by dspacejs** [comment]
@geigi no problem. I found a temporary workaround. I unmounted the drive, opened Cozy, waited for the scan to remove the books. Then closed Cozy, remounted the drive, opened Cozy again and reimported them.

> Have you made sure you have added the storage location only once in settings? I'm not sure what happens when you add the same location twice.

Yep only once.

> What would help with debugging is having a look at Cozy's database

Unfortunately I don't have the issue anymore. Not sure if I'll be able to reproduce it exactly. I remember there being some glitchy behaviour and I can't recall exactly what caused it.
